cordapp/workflows/src/test/java/com/r3/conclave/cordapp/sample/host/ReverseStringEnclaveTest.java

```diff
@@ -66,9 +66,18 @@ public void reverseStringAsAnonymous() throws ExecutionException, InterruptedExc
     }
 
     private String getConstraint() {
-        String mode = System.getProperty("enclaveMode");
-        if (mode == null || !mode.toLowerCase().equals("mock"))
-            return constraint;
-        return mock_constraint;
+        String mode = System.getProperty("enclaveMode").toLowerCase();
+
+        if (mode == null || !mode.toLowerCase().equals("mock")) {
```

Comment:
This is also buggy. Will fix on Monday. But I can't get an environment setup for this test in 1.8, this is a real concern.